### PR TITLE
Set minimum version to PHP 7.0.3

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue # .

### CODE REVIEW

**Comment by sozzled** [comment]
Most people are running with the same PHP minimum version required for Joomla.  Setting the minimum version of PHP to 7.0.3 will discourage people from upgrading to K 5.0.

In other words, requiring people to use PHP 7.0.3 will turn people away from the Kunena project.

Bad idea.

**Comment by 810** [comment]
@sozzled Older php 7 versions has security leaks, so we move to  min php to 7.0.3

**Comment by xillibit** [comment]
That's not prevent to use Kunena on php 5.4.x, 5.5.x or 5.6 series, the title is a bit incorrect

File: src/administrator/components/com_kunena/install/install.script.php [hunk 1]
```diff
@@ -17,11 +17,11 @@ class Com_KunenaInstallerScript
 {
 	protected $versions = array(
 		'PHP' => array (
-			'7.0' => '7.0.4',
+			'7.0' => '7.0.3',
 			'5.6' => '5.6.8',
 			'5.5' => '5.5.13',
 			'5.4' => '5.4.13',
-			'0' => '5.4.35' // Preferred version
```
Comment (coder4life) [inline src/administrator/components/com_kunena/install/install.script.php:24]: I disagree with this changed. Preferred version is the minimum number that the software should run on. The software will throw warnings everytime you try to install on any 5.6, 5.6, and 5.4 version of the software.

Comment (810) [inline src/administrator/components/com_kunena/install/install.script.php:24]: That's correct for me, the users needs some time to update their environments, if we didn't notice them they we be mad. Now we can say, we told you some time to update your environment.

Comment (coder4life) [inline src/administrator/components/com_kunena/install/install.script.php:24]: Although that sets a precedent on what the person should be running versus what is actually needed. Just an FYI most shared hosts where a majority of Kunena installs will be running on do not support PHP 7 yet. Many are pushing to 5.5 and 5.6. The preferred version should focus on two things: 1. the minimum version Kunena is absolutely required to run on 2. the End of Life conditions for older PHP versions. I believe on most frameworks PHP 5.5 is the minimum version required.

Comment (810) [inline src/administrator/components/com_kunena/install/install.script.php:24]: You only get this notice when you have <5.3.10 & 5.4.0 < 5.4.13 ect they have all low numbers. So you should not see this error, only on bad hosts, then you need to switch immediately

**Pull request merged**